### ISSUE TITLE
[VISUAL] $Bumpmap bound to the wrong Sampler in LightmappedGeneric_dx9_helper.cpp

### ISSUE THREAD

**Issue by WhiteRedDragons** [mapbase-source/source-sdk-2013#256]
### Describe the bug
Mistake in LightmappedGeneric_dx9_helper.cpp causes $bumpmap2 to be bound to the wrong sampler

LightmappedGeneric_dx9_helper.cpp Line 450 - SHADER_SAMPLER6 is enabled for $Bumpmap2
LightmappedGeneric_dx9_helper.cpp Line 651 - SHADER_SAMPLER6 receives $Bumpmap2
LightmappedGeneric_ps2_3_x.h Line 151 - **s5 is being designated for $bumpmap2, not s6**

### Solution
change SHADER_SAMPLER6 to SHADER_SAMPLER5
at Line 450 and Line 651 in LightmappedGeneric_dx9_helper .cpp

---

### Additional context
Probably related to many other graphical issues that might have been reported in the past.
Cief reports this shader behaviour also happens on stock shaders

**Comment by Petercov** [comment]
Lines 450 and 651 in LightmappedGeneric_dx9_helper.cpp  are for the flashlight shader. SDK_lightmappedgeneric_flashlight_ps2x.fxc line 61 loads NormalMap2 from sampler 6. There was no issue here.

**Comment by WhiteRedDragons** [comment]
You are correct, we realised this on the map labs discord, I will adjust the issue's name.
Now it baffles me as I cannot find a reason for $bumpmap2 to be broken then
